src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlClientDiagnosticListenerExtensions.cs

```diff
@@ -52,6 +52,7 @@ public static Guid WriteCommandBefore(this DiagnosticListener @this, SqlCommand
                         Operation = operation,
                         ConnectionId = sqlCommand.Connection?.ClientConnectionId,
                         Command = sqlCommand,
+                        TransactionId = transaction?.InternalTransaction?.TransactionId,
```

Comment:
These might be able to be simplified to just:
```suggestion
                        transaction?.InternalTransaction?.TransactionId,
```

Reply:
Done!